### PR TITLE
Add secondary index integration test

### PR DESCRIPTION
**Pull request by brfrn169**
This PR adds secondary index related integration tests. The integration tests check if all data types for secondary indexes are handled correctly. Please take a look!

### CODE REVIEW

**Review by thongdk8 — APPROVED** [review]
LGTM! Thank you!

File: core/src/main/java/com/scalar/db/storage/common/checker/OperationChecker.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public void check(Get get) throws ExecutionException {
     checkProjections(get, metadata);
 
     if (Utility.isSecondaryIndexSpecified(get, metadata)) {
-      if (!new ColumnChecker(metadata, true).check(get.getPartitionKey().get().get(0))) {
+      if (!new ColumnChecker(metadata, false).check(get.getPartitionKey().get().get(0))) {
```
Comment (feeblefakie) [inline core/src/main/java/com/scalar/db/storage/common/checker/OperationChecker.java:39]: Was it a bug?

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

File: core/src/main/java/com/scalar/db/storage/common/checker/OperationChecker.java (hunk 1 above)
Comment (brfrn169) [reply core/src/main/java/com/scalar/db/storage/common/checker/OperationChecker.java:39]: Yes, I found we needed to accept empty values here, that's a very rare case though.

**Pull request merged**